peps/pep-0772.rst

```diff
@@ -245,14 +245,18 @@ Conflicts of interest
 ---------------------
 
 No more than two Packaging Council members should be employed by or
-significantly affiliated with the same entity. An entity would be a company,
-a company and its subsidiaries, or another incorporated entity such as a
-non-profit or educational institution with its own mission and goals.  PSF members are explicitly exempt from this restriction, but _not_ PSF employees, directors, or officers.
+significantly affiliated with the same entity. An entity would be a company, a
+company and its subsidiaries, or another incorporated entity such as a
+non-profit or educational institution with its own mission and goals. PSF
+members are explicitly exempt from this restriction, but _not_ PSF directors,
+or officers.
 
 While we trust council members to act in the best interests of Python rather
 than themselves or their employers, the mere appearance of any one company
 dominating Python development could itself be harmful and erode trust.
 
+PSF staff members are not permitted to be on the Packaging Council.
```

Comment:
Is there a 'cooling-off' period for which this prohibition still applies, e.g. 6-12 months after ending employment?

Wording suggestion:
```suggestion
PSF staff members are not permitted to be members of the Packaging Council.
```

Reply:
> Is there a 'cooling-off' period for which this prohibition still applies, e.g. 6-12 months after ending employment?

We hadn't considered this, but I think it's not necessary.  If someone wants to run for the Packaging Council bad enough to resign from PSF employment, then that shows a pretty significant commitment, especially since they'd still have to run for the council and be elected.

Are you worried about someone who runs for the council, gets elected, and *then* resigns from the PSF?  That seems like a pretty unlikely scenario, and even so, I'm not sure what the problem would be with that.